Cargo.toml

```diff
@@ -39,10 +39,10 @@ fern-logger = { version = "0.5.0", default-features = false }
 
 [features]
 # only default for now, should later be replaced by stronghold since stronghold is more secure
-default = [ "mnemonic", "storage", "stronghold" ]
+default = [ "mnemonic", "storage", "rocksdb", "stronghold" ]
 ledger_nano = [ "iota-client/ledger_nano" ]
 stronghold = [ "iota-client/stronghold" ]
-storage = [ "rocksdb" ]
+storage = []
```

Comment:
Why not keep rocksdb here, if we have to keep it (at least for now)?

Reply:
If we keep it, we couldn't be able to say that we want storage but not rocksdb when compiling for wasm

Reply:
But the way it's in this PR, we also can't use it without default features but with the storage feature, because rocksdb isn't added as new feature?

Reply:
any optional dependency automatically is a feature, so you can just use it like normal

Reply:
Ah nice, I had forgotten that